rule.raml

```diff
@@ -0,0 +1,364 @@
+                schema: batch
+                  example: |
+                    [
+                      {
+                        "href":"",
+                        "rep": {
+                          "rule": "(switch:value = true) and (temperature:temperature >= set-temp:temperature)",
+                          "ruleresult": false,
+                          "ruleenable": false,
+                          "actionenable": false
+                        }
+                      },
+                      {
+                        "href": "modeswitch",
+                        "rep": {
+                          "value": false
+                        }
+                      },
+                      {
+                        "href": "temperature",
+                        "rep": {
+                          "temperature": 23
+                      }
+                      },
+                      {
+                        "href": "set-temperature",
+                        "rep": {
+                          "temperature": 30
+                        }
+                      },
+                      {
+                        "href": "scenevalue",
+                        "rep": {
+                          "lastscene": "heat-off"
+                        }
+                      }
+                    ]
+
+
+/RuleURI/resURI?if=oic.if.create:
+    description: |
+      Create a rule input item using POST with oic.if.create, URI in the query parameter.
+      Creates a new resource in the collection and a link pointing to it.
+    displayName: rule-create
+
+    post:
+      description: |
+        Create interface of the rule resource type
+
+      queryParameters:
+        href:
+          type: string
+
+      body:
+        application/json:
+          schema: create
+            example: |
+              [
+                {
+                  "rt": "oic.r.switch.binary",
+                  "if": ["oic.if.a", "oic.if.baseline" ],
+                  "title":  "Heat Mode"
+                  "rep": {
+                    "rt": "oic.r.switch.binary",
+                    "if": ["oic.if.a", "oic.if.baseline" ]
+                    "value": false
+                  }
+                }
+              ]
+
+        responses:
+          200:
+            body:
+              application/json:
+                schema: create-response
+                  example: |
+                    [
+                      {
+                        "href": "modeswitch",
+                        "rt": ["oic.r.switch.binary"],
+                        "if": ["oic.if.a", "oic.if.baseline" ],
+                        "title":  "Heat Mode"
+                        "rep": {
+                          "rt": ["oic.r.switch.binary"],
+                          "if": ["oic.if.a", "oic.if.baseline" ]
+                          "value": false
+                        }
+                      }
+                    ]
+
+
+/RuleURI?if=oic.if.delete:
```

Comment:
Wouldn't this DELETE request remove the /RuleURI Resource and not the /rules/head-on/fanmode Resource as shown in Line 299?